test/cli/packager_did_you_mean/test.out

```diff
@@ -0,0 +1,4 @@
+__package.rb:6: Unable to resolve constant `ProjectWithLongDiscernableName` https://srb.help/5002
+     6 |  export ProjectWithLongDiscernableName::Foo::Example
+                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Is this the case that you mentioned in your PR description?

Reply:
Yeah so like we could try to say "oh it looks like the suggested symbol is `<PackageSpecRegistry>::ProjectWithLongDiscernableName` and the current symbol is `<root>` + `ProjectWithLongDiscernableName`, so we should try to see if the suggested symbol is "what the user wrote minus `<PackageSpecRegistry>::` but the way to do that while operating on SymbolRefs and NameRefs (instead of string source equality) is just an annoying loop that has a lot of code for what can be done with string comparison in 2 lines (and maybe even be just as fast, idk i didn't test it)